app/vep/models/upload_vcf_files.py

```diff
@@ -92,8 +92,6 @@ async def stream(self):
         except (MaxBodySizeException, ValidationError):
             shutil.rmtree(self.temp_dir)
             raise MaxBodySizeException(MAX_FILE_SIZE)
-        except (Exception, ClientDisconnect) as e:
-            print(e)
-            logging.info(e)
+        except (ClientDisconnect, Exception):
```

Comment:
Could you please add the logging here?

Reply:
This exception is already logged by the caller ([on this line](https://github.com/Ensembl/ensembl-web-tools-api/blob/14cf9ea0c002388cfa2b5b92c4d4b13b0b903d72/app/vep/vep_resources.py#L91)), so adding another logging here would only add noise. Is there a benefit/purpose for adding logging here or would it be better to add this in the future when such a need arises? And in case we add logging here, should we also add it to the MaxBodySizeException/ValidationError above (also, I was wondering why is ValidationError reraised as MaxBodySizeException)?